### ISSUE TITLE
[PRO]: : Issue with Details Extension Triggering Unwanted Form Validation

### ISSUE THREAD

**Issue by PhongLi** [ueberdosis/tiptap#4384]
### Which packages did you experience the bug in?

@tiptap-pro/extension-details": "^2.2.1"

### What Tiptap version are you using?

2.0.4

### What’s the bug you are facing?

I encountered a problem while using the details extension in Tiptap. When clicking the button to toggle the summary of the details, it triggers an event that unintentionally leads to the submission and validation of the form. All other extensions function normally and do not trigger any unintended events.

### What browser are you using?

Chrome

### What did you expect to happen?

Clicking the button to toggle the details extension's summary should not trigger any events that result in form submission or validation.

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by yusren** [comment]
If using Vue3, just add `@submit.prevent`
```ts
<script setup lang="ts">
import TipTap from '@/components/reusable/TiptapEditor.vue'
</script>
<template>
...
<form class="w-full mt-3" @submit.prevent="onSubmit">
...
  <TipTap/>
</form>
...
<template>
```

**Comment by bdbch** [comment]
Thanks. Should be fixed in v2.3.1
